examples/fabric/v4/portConnectivity/ibm/ibm2/README.md

```diff
@@ -30,6 +30,12 @@ At minimum, you must set below variables in `terraform.tfvars` file:
 `seller_asn` - Seller ASN Number
 `seller_region` - Seller Region
 
+## Note
```

Comment:
Nice note here. Would you add this to the published docs as well? Just like where we will start putting the examples to customers that will be shown on the Hashicorp Documentation.

Reply:
The note here was still great. The movement of it to the docs excluded the information about it being the IBM side though.